pyriemann_qiskit/utils/docplex.py

```diff
@@ -201,114 +203,110 @@ class pyQiskitOptimizer:
     def __init__(self):
         pass
 
-    """Hook to apply some transformation on a covariance matrix.
-
-    Parameters
-    ----------
-    covmat : ndarray, shape (n_features, n_features)
-        The covariance matrix.
-
-    Returns
-    -------
-    transformed_covmat : ndarray, shape (n_features, n_features)
-        A transformation of the covariance matrix.
-
-    Notes
-    -----
-    .. versionadded:: 0.0.2
-    """
-
     def convert_covmat(self, covmat):
+        """Hook to apply some transformation on a covariance matrix.
+
+        Parameters
+        ----------
+        covmat : ndarray, shape (n_features, n_features)
```

Comment:
Like in pyRiemann, try to replace `covmat` inputs by `X`, more `sklearn`istic.

And try to generalize covariance matrices to SPD matrices.